src/api/Controllers/LithostratigraphyController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using BDMS.Authentication;
+using BDMS.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BDMS.Controllers;
+
+[ApiController]
+[Route("api/v{version:apiVersion}/[controller]")]
+public class LithostratigraphyController : BdmsControllerBase<LithostratigraphyLayer>
+{
+    private readonly BdmsContext context;
+
+    public LithostratigraphyController(BdmsContext context, ILogger<LithostratigraphyLayer> logger)
+        : base(context, logger)
+    {
+        this.context = context;
+    }
+
+    /// <summary>
+    /// Asynchronously gets the <see cref="LithostratigraphyLayer"/>s of the specified <paramref name="stratigraphyId"/>.
+    /// </summary>
+    /// <param name="stratigraphyId">The id of the stratigraphy referenced in the lithostratigraphy to get.</param>
+    [HttpGet]
+    [Authorize(Policy = PolicyNames.Viewer)]
+    public async Task<IEnumerable<LithostratigraphyLayer>> GetAsync([FromQuery] int stratigraphyId)
```

Comment:
Q: Ist es einfach eine Verbesserung gegenüber der Chrono, dass hier der Get nur mit stratigraphyId möglich ist, oder gibt es da einen technischen Grund?

Reply:
Das ist eine Verbesserung gegenüber der Chrono. Vielleicht wäre es gut die verschiedenen Controller mit einer gemeinsamen Basisklasse zu vereinheitlichen. Das möchte ich aber nicht in diesem PR erledigen.